techniques/systemSettings/userManagement/userManagement/7.1/metadata.xml

```diff
@@ -91,7 +92,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       </SELECT1>
       <SELECT1>
         <NAME>USERGROUP_USER_PASSWORD_POLICY</NAME>
-        <DESCRIPTION>How often do you want to want to check the password</DESCRIPTION>
+        <DESCRIPTION>Password check frequency</DESCRIPTION>
```

Comment:
Given the options (once vs everytime) I'd name this 'policy' rather than 'frequency'